src/backend/common/auth/roles.py

```diff
@@ -15,14 +16,15 @@
 default_roles = [
     (0, SUPER_ADMINISTRATOR, 0),
     (1, ADMINISTRATOR, 1),
-    (2, SITE_MODERATOR, 2),
-    (3, EVENT_ADMIN, 3),
-    (4, EVENT_MOD, 4),
-    (5, SUPPORT_STAFF, 5),
-    (6, SITE_SUPPORTER, 6),
+    (2, SITE_MODERATOR, 3),
```

Comment:
We shouldn't change any of the hierarchies since it won't automatically update the db, I think just keep the hierarchies as before and just have moderator and super moderator both at level 2